src/components/NFT.tsx

```diff
@@ -54,14 +62,12 @@ const AssetCard = (props: { asset: Asset }) => {
   );
 };
 
-
 export const AssetLibrary = () => {
-  const { chain } = useNetwork();
   const [assets, setAssets] = useState<Asset[]>([]);
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { t } = useTranslation('common');
   const [loading, setLoading] = useState(false);
-  const { address, connector, isConnected } = useAccount();
+  const { address, connector, isConnected, chain } = useAccount();
```

Comment:
Also, the `connector` and `isConnected` variables declared on line 23 are not used.